package.json

```diff
@@ -186,18 +229,49 @@
         "category": "Daffodil Debug",
         "enablement": "!inDebugMode",
         "icon": "$(debug-configure)"
+      },
+      {
+        "command": "toggle.experimental",
+        "title": "Enable Experimental Features",
+        "category": "Daffodil Debug"
+      },
+      {
+        "command": "omega_edit.version",
+        "title": "Omega Edit Ω Version Info",
+        "category": "OmegaEdit",
+        "enablement": "experimentalFeaturesEnabled"
+      },
+      {
+        "command": "data.edit",
+        "title": "Data Editor",
+        "category": "OmegaEdit",
+        "enablement": "experimentalFeaturesEnabled"
+      },
+      {
+        "command": "position.goto",
+        "title": "Go to position",
+        "category": "OmegaEdit",
+        "enablement": "experimentalFeaturesEnabled"
+      }
+    ],
+    "keybindings": [
+      {
+        "command": "position.goto",
+        "key": "ctrl+alt+g",
+        "mac": "ctrl+cmd+g",
+        "when": "editorTextFocus"
       }
     ],
     "breakpoints": [
       {
-        "language": "xml"
+        "language": "dfdl"
       }
     ],
     "debuggers": [
       {
         "type": "dfdl",
         "languages": [
-          "xml"
+          "dfdl"
```

Comment:
So in a way you can somewhat import xml extension and go based off that of that, which is what we were doing with snippets previously. However, there are limitations to what you can do with that as you can’t change, add or update certain aspect and it also becomes confusing with auto completion as you will have duplicate items as one will be for xml and one for dfdl. These are the main reasons we went about things this way as it gives us more flexibility with how the language extension works that we won’t get if we just base off xml. It does duplicate things from xml but again we were okay with this because in the long run it will be better for dfdl language to be as flexible as possible as well as making sure the Apache community is in full control of how the language extension functions.

Reply:
Understood. That's fine for now, but I think in general we need to be very careful about what we copy in, and it should be avoided as much as possible. Every piece of some other extension we copy in is just one more thing that we need to maintain, and that just isn't a long term viable solution if it happens to many times and if the pieces are too big. I don't want to have to maintain multiple XML extensions with one line tweaks to work with DFDL.